include/real/real.hpp

```diff
@@ -303,13 +311,15 @@ namespace boost {
                     switch (this->_real_ptr->_kind) {
 
                         case KIND::EXPLICIT:
-                                this->_explicit_it = this->_real_ptr->_explicit_number.cbegin();
-                            this->approximation_interval = this->_explicit_it.approximation_interval;
+                                this->diff_it = std::get<real_explicit>(
```

Comment:
Be consistent in the use of the namespaces, When you defined the diff_it (line 109) you used the full namespace path boost::real::real_explicit. Do the same everywhere.